app/backend/src/couchers/migrations/versions/a98d5e645719_new_profile_fields.py

```diff
@@ -0,0 +1,86 @@
+"""new profile fields
+
+Revision ID: a98d5e645719
+Revises: 6507ac862688
+Create Date: 2021-02-16 23:19:12.390108
+
+"""
+import geoalchemy2
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a98d5e645719"
+down_revision = "6507ac862688"
```

Comment:
Please update to latest head before merge

Reply:
How do you do that?

Reply:
I'll fix it. But I mean to just get the latest revision ID and make the down_revision that. That's often all that's needed to get the migrations linear unless there's clashes.